autorest/codegen/templates/lro_operation_helper.jinja2

```diff
@@ -25,16 +26,17 @@
 {% endif %}
 :keyword callable cls: A custom type or function that will be passed the direct response
 :keyword str continuation_token: A continuation token to restart a poller from a saved state.
-:keyword polling: True for ARMPolling, False for no polling, or a
- polling object for personal polling strategy
-:paramtype polling: bool or ~azure.core.polling.{{ "Async" if async_mode else "" }}PollingMethod
+{% set default_polling_method = "ARMPolling"if code_model.options['azure_arm'] else "LROBasePolling" %}
```

Comment:
Main point of this PR (sorry there's a lot of fluff where I add a keyword to dedup code in the LRO world). Before, we would always generate `ARMPolling `as the default polling method, even when arm flag wasn't set